api/bootstrapper.go

```diff
@@ -18,12 +18,13 @@ func (b Bootstrapper) Bootstrap(ctx map[string]interface{}) error {
 		return fmt.Errorf("config not initialised")
 	}
 
-	registry, ok := ctx[documents.BootstrappedRegistry].(*documents.ServiceRegistry)
+	// just check to make sure that registry is initialised
+	_, ok = ctx[documents.BootstrappedRegistry].(*documents.ServiceRegistry)
```

Comment:
Does it makes sense to let the register services (place where it is actually read) fail if not set then?